.gitignore

```diff
@@ -4,3 +4,4 @@
 *.tfstate.backup
 *.lock.hcl
 lambdas/*.zip
+*.idea
```

Comment:
It's better to add this in your personal `.gitignore`, because otherwise we'd end up adding every editor's file patterns to every repo's `.gitignore`, and we have 60+ repos. (This is why there are no other editor file pattern in there.)

`git config --global core.excludesfile ~/.gitignore_global`